src/au3audio/CMakeLists.txt

```diff
@@ -28,6 +28,6 @@ include(${CMAKE_CURRENT_LIST_DIR}/../au3wrap/au3defs.cmake)
 
 set(MODULE_INCLUDE ${AU3_INCLUDE})
 set(MODULE_DEF ${AU3_DEF})
-set(MODULE_LINK au3wrap)
+set(MODULE_LINK au3wrap trackedit)
```

Comment:
Why was it necessary to link this module?
This looks unexpected, it is alarming that the audio module depends on the track editing module